test/e2e/rte/rte_test.go

```diff
@@ -280,17 +262,17 @@ var _ = ginkgo.Describe("with a running cluster with all the components", func()
 	})
 
 	ginkgo.It("[rte][podfingerprint] should expose the pod set fingerprint status on each worker", func() {
-		nropObj, err := nropcli.NUMAResourcesOperators().Get(context.TODO(), objectnames.DefaultNUMAResourcesOperatorCrName, metav1.GetOptions{})
-		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		nropObj := &nropv1.NUMAResourcesOperator{}
+		err := clients.Client.Get(context.TODO(), client.ObjectKey{Name: objectnames.DefaultNUMAResourcesOperatorCrName}, nropObj)
```

Comment:
nit: missed error handling/reporting here

Reply:
thanks, fixed! even though is a backport, we need 4.12-specific changes anyway, so let's take the chance and add it.